recipes/libftdi/1.x/patches/0001-cmake-targets.patch

```diff
@@ -0,0 +1,787 @@
+@@ -97,7 +115,7 @@ if ( WIN32 )
+   set ( CPACK_NSIS_MODIFY_PATH             ON )
+ endif ()
+ 
+-set(CPACK_RESOURCE_FILE_LICENSE        ${CMAKE_SOURCE_DIR}/LICENSE)
++set(CPACK_RESOURCE_FILE_LICENSE        ${CMAKE_CURRENT_SOURCE_DIR}/LICENSE)
+ 
+ set(CPACK_SOURCE_GENERATOR             TGZ)
+ set(CPACK_SOURCE_IGNORE_FILES          "\\\\.git;~$;build/")
+@@ -136,7 +154,7 @@ endif ()
+ 
+ add_subdirectory(src)
+ if ( FTDIPP )
+-  project(libftdipp1 C CXX)
++  project(libftdi1 C CXX)
+   add_subdirectory(ftdipp)
+ endif ()
+ if ( PYTHON_BINDINGS )
+@@ -150,6 +168,9 @@ if ( EXAMPLES )
+ endif ()
+ add_subdirectory(packages)
+ if ( BUILD_TESTS )
++  project(libftdi1 C CXX)
++  find_package(Boost COMPONENTS unit_test_framework REQUIRED)
++  enable_testing()
+   add_subdirectory(test)
+ endif ()
+ 
+@@ -165,8 +186,8 @@ if(${WIN32})
+    set(libdir      ${CMAKE_INSTALL_PREFIX}/bin)
+ endif(${WIN32})
+ 
+-configure_file(${CMAKE_SOURCE_DIR}/libftdi1.pc.in ${CMAKE_BINARY_DIR}/libftdi1.pc @ONLY)
+-configure_file(${CMAKE_SOURCE_DIR}/libftdipp1.pc.in ${CMAKE_BINARY_DIR}/libftdipp1.pc @ONLY)
++configure_file(${CMAKE_CURRENT_SOURCE_DIR}/libftdi1.pc.in ${CMAKE_BINARY_DIR}/libftdi1.pc @ONLY)
++configure_file(${CMAKE_CURRENT_SOURCE_DIR}/libftdipp1.pc.in ${CMAKE_BINARY_DIR}/libftdipp1.pc @ONLY)
+ install(FILES ${CMAKE_BINARY_DIR}/libftdi1.pc ${CMAKE_BINARY_DIR}/libftdipp1.pc
+         DESTINATION lib${LIB_SUFFIX}/pkgconfig)
+ 
+@@ -202,30 +223,25 @@ set ( LIBFTDI_VERSION_MINOR ${MINOR_VERSION} )
+ 
+ set ( LIBFTDI_USE_FILE ${CMAKE_INSTALL_PREFIX}/${LIBFTDI_CMAKE_CONFIG_DIR}/UseLibFTDI1.cmake )
+ 
+-if(CMAKE_VERSION VERSION_LESS 2.8.8) 
+-  configure_file ( cmake/LibFTDI1Config.cmake.in ${CMAKE_CURRENT_BINARY_DIR}/LibFTDI1Config.cmake @ONLY )
+-  configure_file ( cmake/LibFTDI1ConfigVersion.cmake.in ${CMAKE_CURRENT_BINARY_DIR}/LibFTDI1ConfigVersion.cmake @ONLY )
+-else ()
+-  include(CMakePackageConfigHelpers)
+-
+-  configure_package_config_file (
+-    cmake/LibFTDI1Config.cmake.in
+-    ${CMAKE_CURRENT_BINARY_DIR}/LibFTDI1Config.cmake
+-    INSTALL_DESTINATION ${LIBFTDI_CMAKE_CONFIG_DIR}
+-    PATH_VARS
+-      LIBFTDI_USE_FILE
+-      LIBFTDI_ROOT_DIR
+-      LIBFTDI_INCLUDE_DIR
+-      LIBFTDI_INCLUDE_DIRS
+-      LIBFTDI_LIBRARY_DIRS
+-    NO_CHECK_REQUIRED_COMPONENTS_MACRO
+-  )
+-  write_basic_package_version_file (
+-    LibFTDI1ConfigVersion.cmake 
+-    VERSION ${LIBFTDI_VERSION_STRING} 
+-    COMPATIBILITY AnyNewerVersion
+-  )
+-endif ()
++include(CMakePackageConfigHelpers)
++
++configure_package_config_file (
++cmake/LibFTDI1Config.cmake.in
++${CMAKE_CURRENT_BINARY_DIR}/LibFTDI1Config.cmake
++INSTALL_DESTINATION ${LIBFTDI_CMAKE_CONFIG_DIR}
++PATH_VARS
++  LIBFTDI_USE_FILE
++  LIBFTDI_ROOT_DIR
++  LIBFTDI_INCLUDE_DIR
++  LIBFTDI_INCLUDE_DIRS
++  LIBFTDI_LIBRARY_DIRS
++NO_CHECK_REQUIRED_COMPONENTS_MACRO
++)
++write_basic_package_version_file (
++LibFTDI1ConfigVersion.cmake
++VERSION ${LIBFTDI_VERSION_STRING}
++COMPATIBILITY AnyNewerVersion
++)
```

Comment:
This seems like an overkill. The `if` is always true in CCI and should not need to be changed

Is this really required?